requirements.txt

```diff
@@ -6,6 +6,10 @@ funcsigs
 Flask==1.1.2
 google-cloud-tasks==1.5.0
 google-cloud-ndb
+google-cloud-logging==1.14.0
+
+# Required by google-cloud-logging, https://cloud.google.com/appengine/docs/standard/python/migrate-to-python3/migrate-to-cloud-logging#install
+googleapis_common_protos
```

Comment:
When I run `npm run deps`, I get an error on this line because it duplicates `googleapis-common-protos==1.52.0` on line 15.     I see that on GAE, if I view source for the version of the app that you deployed, there is no lib/google/cloud/logging directory.  So, I think that this line would allow you to successfully run `npm run deps`, which should add that directory locally and then it would be on the server after you run `npm run staging` again.

Reply:
yea I didn't have that line initially and thats what I was confused about.  In the document it requires `googleapis_common_protos`. I will remove that line and try again